### ISSUE TITLE
Document a design for beta.nsf.gov/Drupal login/admin traffic to go through the NSF TIC

### ISSUE THREAD

**Issue by mtboesch** [18F/nsf#139]
## User story
As a technical team member, I want to understand and be able to share with NSF DIS how we are going to ensure that login and secure functionality on beta.nsf.gov/Drupal will go through the NSF TIC so that we can meet ATO requirements and ensure compliance. 

## Definition of done
- [ ] Design is documented and required modules and configurations are identified
- [ ] Implement a POC on cloud.gov
- [ ] Design reviewed by NSF DIS and they have no further questions

**Comment by mtboesch** [comment]
Would the Drupal Restrict IP module be useful for this or would we do this with some functionality that cloud.gov provides for restricting domains/IPs (will this be ready/available in time?) and integrate that into the application?

**Comment by fureigh** [comment]
@mtboesch Yes, restrict_by_ip looks like a solid candidate. The [Drupal 8 version](https://www.drupal.org/node/2570505) appears to allow for restricting login by IP address and also forcing logout when users are outside a specified IP address range. I'd like to test it early next sprint.

**Comment by fureigh** [comment]
(To elaborate a little further: we don't need role-specific restrictions because in this case, we want anyone with any kind of account to have to authenticate from a specified IP range. So that module's existing functionality should suffice.)

**Comment by OpenGlobe** [comment]
This appears to be a duplicate of #145 - I'm going to close this one in preference for the latter.

**Comment by cmc333333** [comment]
@OpenGlobe this is slightly different than #145, which focuses on cloud.gov users (i.e. devs/devops people) and requires some tweaks on the cloud.gov site.

This issue is about the Drupal app's authentication (content admins).

**Comment by OpenGlobe** [comment]
Ah, thank you!

**Comment by OpenGlobe** [comment]
NSF wants to understand the process and have functionality available to be turned on (note from 03/30/18)
